### ISSUE TITLE
Install problem with react versions

### ISSUE THREAD

**Issue by sarahk** [chenjiahan/rodal#65]
New to react so there may be an easy workaround 
```
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"17.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@">= ^16.0.0" from rodal@1.8.3
npm ERR! node_modules/rodal
npm ERR!   rodal@"*" from the root project
```

**Comment by desunbera** [comment]
i have the same issue

**Comment by namipsg** [comment]
Same issue with react v18.0.0 on Next.js

**Comment by brezhitskiy42** [comment]
same issue on Next.js

**Comment by chenjiahan** [comment]
Fixed in v2.0.0.
